doc/api-extensions.md

```diff
@@ -2407,3 +2407,8 @@ This extension adds a new image restriction, `requirements.nesting` which when `
 
 Adds the {config:option}`instance-miscellaneous:linux.kernel_modules.load` container configuration option. If the option is set to `ondemand`, the `finit_modules()` syscall is intercepted and a privileged user in the container's user namespace can load the Linux kernel modules specified in the
 allow list {config:option}`instance-miscellaneous:linux.kernel_modules`.
+
+## `allocate_external_ips`
+
+Adds the ability to use an unspecified IPv4 or IPv6 address in the `listen_address` field of the request body for `POST /1.0/networks/{networkName}/load-balancers` and `POST /1.0/networks/{networkName}/forwards`.
```

Comment:
Can you add the example of an unspecified IP address in brackets for IPv4 and IPv6 here please.